config/config.go

```diff
@@ -334,6 +341,7 @@ var Default = Config{
 	AnalyticsConfig: AnalyticsConfigConfig{
 		IgnoredIPs: make([]string, 0),
 	},
+	Dns:DnsConfig{EnableCaching:false, TTL: 3600 * 1000, CheckInterval: 3600 * 1000},
```

Comment:
This line isnt neccessary if it's off by default

Reply:
29984086768d9d417ec85294c7d9ec0bfcaeed24
fixed